secret_store/src/key_server_cluster/admin_sessions/key_version_negotiation_session.rs

```diff
@@ -361,8 +391,47 @@ impl<T> SessionImpl<T> where T: SessionTransport {
 		let confirmations = data.confirmations.as_ref().expect(reason);
```

Comment:
Out-of-scope for this PR, would be nice if `fn try_complete` would return Result instead!